tests/steps/given.ts

```diff
@@ -0,0 +1,39 @@
+import { expect } from 'chai';
+import { api } from '../shared/classes/api';
+import { Given, Then } from "@cucumber/cucumber";
+import GPS from '@/models/GPS';
+import ConnectMongoDB from '@/lib/mongodb';
+
+let apiResponseData;
+let databaseData;
+
+Given('I clear the database', async function () {
+    const db = await ConnectMongoDB();
+    await GPS.deleteMany();
+});
+
+Given('I insert GPS data into the database', async function () {
+    const gpsData = {
+    latitude: 49.2243,
+    longitude: 4.5552,
+    speed: 30,
+    heading: 45,
+    };
+
+    await GPS.create(gpsData);
+});
+
+Then('the response data matches the data in the database', async function () {
+        const db = await ConnectMongoDB();
```

Comment:
Can you please test if we still need this `ConnectMongoDB()` call given that we do it on the step where we clear the database?  🤔

Reply:
@JordanChen123 Did the tests still pass when you removed this statement?

Reply:
Yes the tests still passed when I removed the statement. I should've clarified it beforehand that's my bad

Reply:
All good